crates/sui-types/src/deny_list_v2.rs

```diff
@@ -140,38 +141,43 @@ pub fn check_coin_deny_list_v2_during_execution(
             .entry(coin_type.to_canonical_string(false))
             .or_insert_with(BTreeSet::new)
             .insert(owner);
+        num_regulated_transfers += 1;
```

Comment:
why is this incremented here, instead of after line 149?

Reply:
We could, but then we would need to add the size of the owners there. This is a lot simpler.